### DISCUSSION TITLE
Is there a way to download posts with a specific title in a sub reddit?

### DISCUSSION THREAD

**Discussion by SunConure** [MalloyDelacroix/DownloaderForReddit#342] [Q&A]
Hello, I was wondering if that it possible because I want to download posts that only mention Mexico in the title on an artifact subreddit.

**Comment by MalloyDelacroix** [comment]
Unfortunately, this is not possible at this time.  This is a feature that is going to be added in the future.

**Reply by SunConure** [reply]
Ah alright, thanks for the information.
